.github/workflows/ci-checks.yml

```diff
@@ -26,9 +26,12 @@ jobs:
       run: yarn install --frozen-lockfile
     - run: yarn lint
     - run: yarn build
-    - run: yarn test
-    - run: yarn start
-    - run: yarn cypress:test
+    - name: Client unit test
+      run: yarn test
+    - name: Start client in background
+      run: yarn start &
```

Comment:
We have to run this command in background to prevent it from blocking next command ?

Reply:
correct